src/MultiSelectAutocomplete/MultiSelectAutocomplete.jsx

```diff
@@ -0,0 +1,192 @@
+/**
+ * EXPERIMENTAL
+ * 
+ * If this replaces the current "Autocomplete" in the future,
+ * the "MultiSelectAutocomplete" should remain an alias or else
+ * it would have to be a major release and not backwards compatible.
+ */
+
+import React, { useRef } from 'react';
+import Select from 'react-select';
+import PropTypes from 'prop-types';
+
+import Readonly from '../Readonly';
+
+import { getTemplates } from '../Autocomplete/Autocomplete.utils';
+
+import { concatClasses } from '../utils/Utils';
+
+import './MultiSelectAutocomplete.scss';
+
+export const DEFAULT_CLASS = 'hods-multi-select-autocomplete';
+
+const DEFAULT_VALUE = { value: "", label: "Select..." };
+const COLORS = {
+  GOVUK_COLOR_BLUE: '#1d70b8',
```

Comment:
This doesn't need the whole `GOVUK_COLOR_` prefix when it's inside of an object like this...
```suggestion
  BLUE: '#1d70b8',
```